package.json

```diff
@@ -59,7 +59,7 @@
     "bcrypt": "^5.0.0",
     "body-parser": "^1.18.3",
     "cezerin2-client": "^0.38.0",
-    "cezerin2-theme": "^1.3.0",
+    "cezerin2-theme": "^1.0.0",
```

Comment:
Why to import theme from theme repo here? And why have you removed the one fullfluid theme from theme repo? The theme repo was established to upload different themes and not just one. Please revert all changes back to fullfluid theme in theme repo. Thnak you!

Reply:
Yeah I was thinking to add different themes as packages to change them. But sure.